accelerator/apis.c

```diff
@@ -285,21 +285,23 @@ status_t api_find_transactions_obj_by_tag(const iota_client_service_t* const ser
   return ret;
 }
 
-status_t api_receive_mam_message(const iota_client_service_t* const service, const char* const chid,
-                                 char** json_result) {
+status_t api_receive_mam_message(const iota_config_t* const tangle, const iota_client_service_t* const service,
```

Comment:
Shall the instance of `iota_config_t *` be named after `tangle`? How about changing it to `conf`?

Reply:
I will rename all the instance of `iota_config_t *` into `conf`